Gems/QtForPython/Code/Source/Platform/Linux/InitializeEmbeddedPyside2.h

```diff
@@ -12,7 +12,9 @@
 
 namespace QtForPython
 {
-    const char* s_libPythonLibraryFile = "libpython3.10.so";
+    // s_libPythonLibraryFile must match the library name listed in (O3DE Engine Root)/python/runtime/.../python-config.cmake
+    // in the set(${MY}_LIBRARY_xxxx sections.
+    const char* s_libPythonLibraryFile = "libpython3.10.so.1.0"; 
```

Comment:
I think this is fine for now, since the other pre-load files here reference the concrete file, not the major version symlink, but the real problem is that the major symlink for this file did not get copied over from the dependencies from the [3P package](https://github.com/o3de/3p-package-source/blob/2af5c1d1e6f44f4183a66d2055ac2874e528b881/package-system/python/linux_x64/python-config.cmake#L49-L54).

Reply:
Yes, I didn't see any mention of it, in the python-config in the python package, which is where such dependencies would be mentioned, I think (?)